docs/source/tutorials/07_using_model_hub_nodes.rst

```diff
@@ -0,0 +1,334 @@
+.. include:: /include/substitution.rst
+
+*********************
+Using Model Hub Nodes
+*********************
+
+PeekingDuck has support for external model hub models. You can leverage the model
+hub nodes to use these external models with the PeekingDuck pipeline. The model
+hub nodes differ in design from traditional model nodes as the model hub models
+perform a variety of computer vision tasks. This tutorial demonstrates how
+to use these nodes through some sample pipelines.
+
+
+List of Model Hub Nodes
+=======================
+
+The table below shows the model hub nodes available.
+
++-----------------------------------------------------------------+---------------------------------+
+| Model                                                           | Documentation                   |
++=================================================================+=================================+
+| `Hugging Face Hub <https://huggingface.co/docs/hub/index>`_     | :mod:`model.huggingface_hub`    |
++-----------------------------------------------------------------+---------------------------------+
+| `MediaPipe Solutions <https://google.github.io/mediapipe/>`_    | :mod:`model.mediapipe_hub`      |
++-----------------------------------------------------------------+---------------------------------+
```

Comment:
Suggest swapping the `Documentation` and `Model` columns, to make the look more consistent with our API documentation.
Also, to rename `Model` as `Model Hubs`.

Reply:
Changed